contributors/devel/sig-release/cherry-picks.md

```diff
@@ -31,37 +31,98 @@ branches.
+     release team members see the [appropriate release folder](https://git.k8s.io/sig-release/releases)
+     for the target release's team contact information. You may cc them with
+     `<@githubusername>` on your cherry-pick PR.
+
    * For prior branches, check the [patch release schedule](https://git.k8s.io/sig-release/releases/patch-releases.md), which includes contact information for the patch release team.
 
-## Cherry-pick Review
+Compared to the normal master branch merges leading up to a release, which
+is typically on the order of 1000 pull requests in 3 months, the release
+branches typically see around 200 pull requests across 9 months. This
+amounts to one or two dozen a month on the stable branches versus many
+hundreds a month on master.  This is because there is an order or two of
+magnitude higher scrutiny.  Again the emphasis is on critical bug fixes.
 
-Cherry-pick pull requests have an additional requirement compared to normal pull
-requests.
-They must be approved specifically for cherry-pick by Approvers.
-The [Branch Manager](https://git.k8s.io/sig-release/release-team/role-handbooks/branch-manager) 
-or the [Patch Release Team](https://git.k8s.io/sig-release/release-team/role-handbooks/patch-release-manager)
-are the final authority on removing the `do-not-merge/cherry-pick-not-approved`
-label  and triggering a merge into the target branch.
-
-Cherry-pick pull requests follow the same release note requirements as
-other pull requests, except the release note stanza will auto-populate from
-the master branch pull request from which the cherry-pick originated.  If
-this is unsuccessful the `do-not-merge/release-note-label-needed` label
-will be applied and the cherry-pick author must edit the pull request
-description to [add a release note](https://git.k8s.io/community/contributors/guide/release-notes.md).
+If you are proposing a cherry pick and it is not a clear and obvious
+critical bug fix, please reconsider.  If upon reflection you wish to
+continue, bolster your case by supplementing your PR with, eg:
+
+ * A GitHub issue detailing the problem
+
+ * Scope of the change
+
+ * Risks of adding a change
+
+ * Risks of associated regression
+
+ * Testing performed, test cases added
+
+ * Key stakeholder SIG reviewers/approvers attesting to their confidence in the
+   change being a required backport
+
+ * If the change is in (to eventually be moved out of core Kubernetes) cloud
+   provider platform code, describe your customer impact, how the
+   issue escaped initial testing, remediation taken to prevent similar
+   future escapes, and why the change cannot be carried in your downstream
+   fork of the Kubernetes project branches.  If you were not engaged in a
+   feature's upstream development and it does not work properly on your
+   platform, it is not the community's responsibility to support your miss by
+   backporting feature enablement.
```

Comment:
I agree with the sentiment, but the personal targeting of the sentence ("If you were not engaged ... to support your miss ...") seems a touch overbearing. Is there a more neutral way to phrase this?

Reply:
Updated words.

Reply:
Am curious if folks think this part reads better now.